controllers/CommentController.py

```diff
@@ -0,0 +1,41 @@
+from flask import Blueprint, jsonify, request
+from dtos.responses import *
+from services import CommentService
+from dtos import CommentDto
+import json
+
+comment_blueprint = Blueprint('comment', __name__)
+
+
+@comment_blueprint.route('/comment', methods=['POST'])
+def postComment() -> (Response, int):
+    req_data: dict = request.get_json()
+    commentService = CommentService()
+    commentDto = CommentDto.fromJson(req_data)
+    response: json = commentService.postComment(commentDto)
+    json_res = json.dumps(response, default=lambda o: o.__dict__)
+    return json_res, response.get('status')
+
+
+@comment_blueprint.route('/comment/p/<int:postId>', methods=['GET'])
```

Comment:
```suggestion
@comment_blueprint.route('/comment/post/<int:postId>', methods=['GET'])
```
I think we decided to not use abbreviations in the routes. That's my bad for doing it in the fist place on `/post`

Reply:
Though with the final demo in a week, I guess it doesn't really matter